source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/ReportDashboard.cshtml

```diff
@@ -16,50 +16,69 @@
+      <div class="content-selector-label">
+        <p>Período<p>
+      </div>
+      <div class="content-selector-dropdown">
+        @Html.DropDownList("selectedPeriod", (IEnumerable<SelectListItem>)ViewBag.ReportPeriods)
+      </div>
     </div>
-    <table class="table">
-        <thead>
-            <tr data-bind="foreach: headers">
-                <!-- ko if: sortable -->
-                <th data-bind="attr: { class: 'col-' + ($index()+1) }, click: $parent.sort, css: { selected: activeSort}">
-                    <i class="icon arrow"></i>
-                    <span data-bind="text: title"></span>
-                </th>
-                <!-- /ko -->
-                <!-- ko if: !sortable -->
-                <th data-bind="attr: { class: 'col-' + ($index()+1) }"><span data-bind="text: title"></span></th>
-                <!-- /ko -->
-            </tr>
-        </thead>
-        <tbody data-bind="foreach: items">
-            <tr>
-                <td data-bind="text: period"></td>
-                <td data-bind="text: fullName"></td>
-                <td data-bind="text: currentPosition"></td>
-                <td data-bind="text: seniority"></td>
-                <td data-bind="text: responsibleId"></td>
-                <td><span class="state-doc" data-bind="text: stateName, attr: { 'class': stateClasses }"></span></td>
-                <td><a class="link-responsable top-tip" data-bind="if: state() == 6, attr: {href: calificationUrl}" data-tips="Ver Evaluación"><i class="icon view"></i><span>Ver Evaluación</span></a></td>
-            </tr>
-        </tbody>
-    </table>
-    <script type="text/javascript">
-        var employeePeriod = "@ViewBag.Period";
-    </script>
+  </div>
+  <table class="table">
+    <thead>
+      <tr data-bind="foreach: headers">
+        <!-- ko if: sortable -->
+        <th data-bind="attr: { class: 'col-' + ($index()+1) }, click: $parent.sort, css: { selected: activeSort}">
+          <i class="icon arrow"></i>
+          <span data-bind="text: title"></span>
+        </th>
+        <!-- /ko -->
+        <!-- ko if: !sortable -->
+        <th data-bind="attr: { class: 'col-' + ($index()+1) }"><span data-bind="text: title"></span></th>
+        <!-- /ko -->
+      </tr>
+    </thead>
+    <tbody data-bind="foreach: items">
+      <tr>
+        <td data-bind="text: period"></td>
+        <td data-bind="text: fullName"></td>
+        <td data-bind="text: currentPosition"></td>
+        <td data-bind="text: seniority"></td>
+        <td class="responsible-column">
+          <span data-bind="if: isEvaluationManager">
+            <a class="calification-links edit top-tip" data-bind="click: showResponsibleManager">
+              <i class="icon edit"></i>
+              <span data-bind="text: responsibleId()"></span>
+            </a>
+          </span>
+          <!-- ko ifnot: isEvaluationManager -->
+          </span><span data-bind="text: responsibleId"></span>
+          <!-- /ko -->
```

Comment:
Oh, I think that there is a dangerous thing here:

```html
<!-- ko ifnot: isEvaluationManager -->
    </span><span data-bind="text: responsibleId"></span>
<!-- /ko -->
```
Here the HTML hierarchy is being broken, my experience with _Knockout_ makes me think that it could be problematic, _Knockout_ parser relies in HTML hierarchy and does not treat it as a simple string, so, close an element dependending on a condition seems to be a bad idea.